configuration.yaml

```diff
@@ -3,6 +3,7 @@
 ## Do not commit changes that contain credentials or sensitive data!
 
 server:
+  rootPath: /api/v0/*
```

Comment:
This change is good, but we need to do a bit more thinking on versions. We have both physical and virtual api endpoints. The versions on these may or may not be the same. Also, the rootpath of the api endpoint should have something else that distinguishes the api, but that's up to each given api.